exo.kernel.component.common/src/main/java/org/exoplatform/services/scheduler/impl/JobSchedulerServiceImpl.java

```diff
@@ -645,6 +664,10 @@ public void start()
    {
       try
       {
+         //Remove the list persisted jobs (jobs removed from the conf)
+         for (JobKey jobKey : persistedJobKeyList ){
+            scheduler_.deleteJob(jobKey);
```

Comment:
No, at start JobSchedulerServiceImpl service , I remove only the list of old  persisted jobs removed from the configuration (persistedJobKeyList contain at start point only the diff between configuration and old persisted list).